msal/build.gradle

```diff
@@ -426,6 +428,10 @@ tasks.whenTaskAdded { task ->
         && !task.name.contains('Snapshot')
         && !task.name.contains('Test')
         && !task.name.contains('Local')) {
-        task.dependsOn 'lint', 'javadocJar', 'sourcesJar'
```

Comment:
Was there a related pipeline update that we should reference?

Reply:
I filed a workitem to fix this in the gradle.  Link above.